cargo-dist/src/init/init_dist_profile.rs

```diff
@@ -0,0 +1,45 @@
+use crate::{config::Config, errors::DistResult, PROFILE_DIST};
+use axoasset::toml_edit;
+
+pub fn init_dist_profile(
```

Comment:
since this is in a folder called init do we want to drop the init here?

Reply:
For things yoinked from `init/v0.rs` (previously `init.rs`), I'm trying to avoid renaming things right now.

Once I'm done splitting things apart, I'm open to renaming stuff.

Reply:
The file can def be renamed to just `init/dist_profile.rs`, though. I'll go ahead and do that.